Assets/Scripts/Battlefield/GridSystem/GridHolder.cs

```diff
@@ -1,25 +1,10 @@
 ﻿using System.Collections;
 using System.Collections.Generic;
 using UnityEngine;
+using SwordAndBored.Battlefield;
 
 public class GridHolder : MonoBehaviour
 {
-    public int size = 50;
-
     public Tile[,] tiles;
-
-    private void Start()
-    {
-        int k = 0;
-        tiles = new Tile[size, size];
-        Tile[] children = GetComponentsInChildren<Tile>();
-        for (int i = 0; i < size; i++)
-        {
-            for (int j = 0; j < size && k < children.Length; j++)
-            {
-                tiles[i, j] = children[k];
-                k++;
-            }
-        }    
-    }
+    
```

Comment:
This reeeaaaallllyyy doesn't need to be a MonoBehavior. It gives me the chills to think something like this could be a component attached to some gameobject instead of just instantiated once and passed through in some Manager.